completions.go

```diff
@@ -616,9 +616,9 @@ $ source <(%[1]s completion bash)
 
 To load completions for every new session, execute once:
 Linux:
-  $ %[1]s completion bash > /etc/bash_completion.d/%[1]s
+$ %[1]s completion bash > /etc/bash_completion.d/%[1]s
```

Comment:
Maybe keep the indentation and add `.. code:: sh` before? See https://docutils.sourceforge.io/docs/ref/rst/directives.html#code

Reply:
Would this be interpreted correctly for the help message @umarcor ?

Reply:
To be honest, I am genuinely confused. To my understanding, the help message is printed to screen/terminal, so unrelated to RST. My previous reply was considering that line 619 is going to end being part of a restructuredtext file. However, I now see why you are asking, since this is indeed the "Long" field of a command. Is the same text source used for both printing the help in screen and for generating the rst docs?

Reply:
It's definitely used for the help message and I'm learning with this PR that it is also used for RST.  So the original idea of removing the indentation is probably the best way forward. What do you think?

Reply:
I think it is technically not correct to remove the indentation, because the resulting RST is not semantically valid. However, given the constraints, I believe it's fair enough. It allows removing the error found by @melissamahoney-mongodb and we don't need to learn the details of doc generation right now. I'll mark this dialogue as resolved, and we'll deal with docgen some other day :wink:.